website/blog/2024-08-07-manipulate-devtools.md

```diff
@@ -0,0 +1,325 @@
+---
+title: "Interacting with Chrome DevTools from WebDriverIO"
+authors: George Romanas & Nikos Lytras
+authorsImageURL: https://avatars.githubusercontent.com/u/48865319?s=400&u=e292306d956ecd6f5d94ff8cde2bc189841c270a&v=4
+                 https://avatars.githubusercontent.com/u/150701199?v=4
+---
+
+# Interacting with Chrome DevTools from WebDriverIO: A Guide
+
+One of WebDriverIO features is the ability to interact with Chrome DevTools, which can be extremely helpful for debugging and testing web applications and browser extensions. This guide will show you how to perform any interaction with DevTools using WebDriverIO v8 and will follow the WebDriverIO v7 guide. In this blog, we'll explore how to use WebDriverIO to interact with the Chrome DevTools, specifically focusing on interacting with the PixiJS extension.
```

Comment:
We will deprecate v7 soon, so I am not sure if it makes sense to highlight this here.

Reply:
done